README.md

```diff
@@ -22,3 +28,5 @@ If you don't want to show it, uncheck `View > Toolbar > Encoding`.
 Cloned from encoding module of main/contrib repository. (Tomas Zezula)
 
 Also see https://netbeans.org/bugzilla/show_bug.cgi?id=114123
+
+Contacto: scherzo16@gmail.com
```

Comment:
Sorry, I would like to avoid adding this because user may misunderstand.
Please write like the following (in Spanish):

Translator : @javatlacati 

Then, please separate to another README.file e.g. `README_es.md`